ov/interconnect.go

```diff
@@ -72,16 +74,6 @@ type Interconnect struct {
 	URI                           utils.Nstring        `json:"uri,omitempty"`                           // "uri": "/rest/interconnects/2b322628-e5a9-4843-b184-08345e7140c3"
 }
 
-type IcmLicence struct {
-	Licence []Licence `json:"licence,omitempty"` // "licence": []
-}
-
-type Licence struct {
-	ConsumedCount int    `json:"consumedCount,omitempty"` // "consumedCount": 0,
-	LicenseType   string `json:"licenseType,omitempty"`   // "licenseType": "Synergy 8Gb FC Upgrade",
-	RequiredCount int    `json:"requiredCount,omitempty"` // "requiredCount": 0,
-	State         string `json:"state,omitempty"`         // "state": "Yes",
-}
```

Comment:
Is there a reason why this struct is removed?

Reply:
This is already in one of the other resources